test/container/Dockerfile

```diff
@@ -37,6 +37,8 @@ ENV GOPATH /go
 ADD ./hack/tool-versions.sh .
 ADD ./hack/install.sh .
 ADD ./hack/installers installers
+ADD go.mod go.mod
+ADD go.sum go.sum
```

Comment:
Thanks @jessesuen for pushing on. Unfortunately this change is not enough as we'll still need to set the PATH to include `/dist`, which is set by [this line in install-codegen-go-tools.sh](https://github.com/argoproj/argo-cd/blob/63e70a9f71915f2988c6117f280ad290eec758cd/hack/installers/install-codegen-go-tools.sh#L19) and holds the binary. 
For reference, I got around the initial issues with [my changes here](https://github.com/ocraviotto/argo-cd/commit/08dbd3f511bfa2b9bd241b9f29885595a3bdf8da).

Reply:
Thanks for the tip. Yes, I think test container should have /dist to PATH as well. I'm trying to test this but the current dockerhub outage is making this impossible. I'm also working through some arm64 specific problems with `make codegen` and I now realize that no one has ever gotten `make codegen` working on arm64.

Reply:
This should be resolved with PATH being set in image